chainio/clients/avsregistry/reader.go

```diff
@@ -220,3 +369,75 @@ func (r *AvsRegistryChainReader) IsOperatorRegistered(
 	registeredWithAvs := operatorId != [32]byte{}
 	return registeredWithAvs, nil
 }
+
+func (r *AvsRegistryChainReader) QueryExistingRegisteredOperatorPubKeys(
+	ctx context.Context,
+	startBlock *big.Int,
+	stopBlock *big.Int,
+) ([]types.OperatorAddr, []types.OperatorPubkeys, error) {
+
+	blsApkRegistryAbi, err := abi.JSON(bytes.NewReader(eigenabi.BLSApkRegistryAbi))
+	if err != nil {
+		r.logger.Error("Error getting Abi", "err", err)
+		return nil, nil, err
+	}
+
+	query := ethereum.FilterQuery{
+		FromBlock: startBlock,
+		ToBlock:   stopBlock,
+		Addresses: []gethcommon.Address{
+			r.blsApkRegistryAddr,
+		},
+		Topics: [][]gethcommon.Hash{{blsApkRegistryAbi.Events["NewPubkeyRegistration"].ID}},
+	}
+
+	logs, err := r.ethClient.FilterLogs(ctx, query)
+	if err != nil {
+		r.logger.Error("Error filtering logs", "err", err)
+		return nil, nil, err
+	}
+	r.logger.Info("logs:", "logs", logs)
+
+	operatorAddresses := make([]types.OperatorAddr, 0)
+	operatorPubkeys := make([]types.OperatorPubkeys, 0)
+
+	for _, vLog := range logs {
+
+		// get the operator address
+		r.logger.Infof("deleteme", "vLog", vLog.Topics)
```

Comment:
delete this?

Reply:
[3bb65cb](https://github.com/Layr-Labs/eigensdk-go/pull/91/commits/3bb65cbacf4639a97ddb1155cf911106976e5e21)